src/towncrier/__init__.py

```diff
@@ -4,29 +4,3 @@
 """
 towncrier, a builder for your news files.
 """
-
-from __future__ import annotations
-
-from incremental import Version
-
-
-__all__ = ["__version__"]
-
-
-def __getattr__(name: str) -> Version:
-    if name != "__version__":
-        raise AttributeError(f"module {__name__} has no attribute {name}")
-
-    import warnings
-
-    from ._version import __version__
-
-    warnings.warn(
-        "Accessing towncrier.__version__ is deprecated and will be "
-        "removed in a future release. Use importlib.metadata directly "
-        "to query for towncrier's packaging metadata.",
-        DeprecationWarning,
-        stacklevel=2,
-    )
-
-    return __version__
```

Comment:
I'm not sure this should be removed. It seems like there's still a bit of software relying on `pkg.__version__` such that I've starte de-deprecating it in my own packages. See also https://github.com/python-attrs/attrs/issues/1136

(to be clear: it has to be ported, not just left around ;))

Reply:
re-added #627